BUILDING.md

```diff
@@ -12,8 +12,127 @@ For building the manual, see the README in the [Manual repository](https://githu
 
 Windows
 -------
+ Visual studio version:
+ 1.) Download:
+   https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017
+
+   Install: Tools for Visual Studio 2019
+     MSVC v142 
+     Windows 10 SDK
+     C++ Cmake Tools for Windows
+     MSVC v141 - VS2017 C++ x64/x86 build Tools
+
+ 2.) Download:
+   http://download.qt.io/official_releases/qt/5.9/5.9.7/qt-opensource-windows-x86-5.9.7.exe
+   Install 
+      MSVC 2017 64-bit
+ 3.) Compile (Console)
+
+   set Qt5_DIR=E:\Qt\Qt5.9.7\5.9.7\msvc2017_64\lib\cmake\Qt5
+   cd {DwarfTherapist Source Dir}
+   mkdir build
+   cd bild
+   cmake -G "Visual Studio 15 2017 Win64" ..\
+   MSBuild.exe DwarfTherapist.sln /p:Configuration=Release 
+
+ 4.) Prepare DT directory
+   Copy from E:\Qt\Qt5.9.7\5.9.7\msvc2017_64\bin
```

Comment:
Use windeployqt instead.

Reply:
Ok, need to learn about this. Currently I don't know anything about it.